docs/rest-api.yml

```diff
@@ -7,29 +7,29 @@ info:
   license:
     name: BSD-3-Clause
 schemes:
-  - http
+  - [http, https]
 securityDefinitions:
   token:
     type: apiKey
     name: Authorization
     in: header
 security:
   - token: []
-basePath: /hub/api/
+basePath: /hub/api
 produces:
   - application/json
 consumes:
   - application/json
 paths:
   /:
     get:
-      summary: See the version of JupyterHub itself
+      summary: Get JupyterHub version
       description: |
-        This endpoint is not authenticated,
-        so clients can identify the version of JupyterHub before setting up authentication.
+        This endpoint is not authenticated. Clients and user can identify
```

Comment:
I would like to preserve the information that it is not authenticated *for the purpose* of this being the case (previously indicated by `so ...`), rather than separate statements.